configuration/configuration_blackbox_test.go

```diff
@@ -141,3 +150,49 @@ func TestGetKeycloakEndpointUserInfoSetByEnvVaribaleOK(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "userinfoEndpoint", url)
 }
+
+func TestGetTokenPrivateKeyFromConfigFile(t *testing.T) {
+	t.Parallel()
+
+	envKey := generateEnvKey(varTokenPrivateKey)
+
+	realEnvValue := os.Getenv(envKey) // could be "" as well.
+
+	os.Unsetenv(envKey)
+	defer os.Setenv(envKey, realEnvValue) // set it back before we leave.
+
+	// env variable NOT set, so we check with config.yaml's value
+
+	viperValue := config.GetTokenPrivateKey()
+	assert.NotNil(t, viperValue)
+
+	parsedKey, err := jwt.ParseRSAPrivateKeyFromPEM(viperValue)
+	require.Nil(t, err)
+	assert.NotNil(t, parsedKey)
+}
+
+func TestGetTokenPublicKeyFromConfigFile(t *testing.T) {
+	t.Parallel()
+
+	envKey := generateEnvKey(varTokenPublicKey)
+	realEnvValue := os.Getenv(envKey) // could be "" as well.
+
+	os.Unsetenv(envKey)
+	defer func() {
+		os.Setenv(envKey, realEnvValue)
```

Comment:
When you overwrite environment variables, the test cannot be marked to be executed in parallel. Please remove `t.Parallel()`.